packages/core/src/Cardano/types/StakePool/primitives.ts

```diff
@@ -12,6 +12,12 @@ export type PoolId = OpaqueString<'PoolId'>;
  */
 export const PoolId = (value: string): PoolId => typedBech32(value, 'pool', 45);
 
+/**
+ * @param {string} value blake2b_224 digest of an operator verification key hash
+ * @throws InvalidStringError
+ */
+export const PoolIdFromKeyHash = (value: string): PoolId => toTypedBech32('pool', value);
```

Comment:
Let's move this out to where PoolId is defined and make this a property of PoolId. Also, `value` can be narrowed down.
```suggestion
export const PoolId.fromKeyHash = (value: Ed25519KeyHash): PoolId => toTypedBech32('pool', value);
```